cmd/fleetctl/gitops_test.go

```diff
@@ -2238,6 +2244,16 @@ func TestGitOpsTeamVPPApps(t *testing.T) {
 				}, nil
 			}
 
+			ds.LabelIDsByNameFunc = func(ctx context.Context, labels []string) (map[string]uint, error) {
```

Comment:
Thinking we need to make sure that all of the expected labels get asked for? Maybe with the equal fn? And check if label IDs by name was invoked if expected labels count > 0? These would be in a post-check when we have a blank desired error.